### PR TITLE
Revamping the collapsibility of the QuestDisplay

### PR DESCRIPTION
**Pull request by RegisCoaxans**
Clicking on the QuestDisplay header now only collapses a daily quest block.
Questline blocks individually collapse when we click on their own respective header.

With more and more questlines, this change helps keep the UI reasonably well-sized and readable while starting all the quests that we want at once.

qUI

### CODE REVIEW

**Comment by Aegyo** [comment]
While we're here, doesn't this list feel a bit cramped to anyone else? How does it look with a small margin between each questDisplayBlock?

**Review by Aegyo — APPROVED** [review]

**Comment by RegisCoaxans** [comment]
There is already a small 1px margin between each block. Do you think that it should be wider ? How much ?
I can not really say as it still looks readable to me. It might also come from the fact that I use a theme which strongly compacts things.

Here are some examples; 1, 3, 5px between blocks :
qUI

**Comment by Aegyo** [comment]
I like the 5px one but I don't really mind if it is changed or not, I'll leave it for Jaaslet to decide

**Comment by Jaaslet** [comment]
All 3 looks good to me... Sorry i am not more helpful :p

**Comment by RedSparr0w** [comment]
I think I like 1px, with 5px margin at the bottom of the progress bars 🤷

**Comment by RegisCoaxans** [comment]
I agree with that last idea, and I think that it looks good. Do I push this change ?
qUI

**Comment by RedSparr0w** [comment]
Looks good to me! 👍

**Comment by RegisCoaxans** [comment]
There we go. Thank you for the advices.

File: src/styles/quest.less [hunk 1]
```diff
@@ -37,15 +37,22 @@
...
-  }
+	
+	table {
+		margin-bottom : 4px;
+	}
 }
+
+.questDisplayBlock:last-child table {
+	margin-bottom : 0px;
+}
```
Comment (RedSparr0w) [inline src/styles/quest.less:58]: I believe this should also work with less, without having to do it separately: 
```suggestion
    }
    &:last-child table {
        margin-bottom : 0px;
    }
}
```

**Comment by RegisCoaxans** [comment]
It does work.

**Comment by RedSparr0w** [comment]
Looks good, thanks for the PR! 👍

**Pull request merged**